docs/advanced/smart_memory_estimation.md

```diff
@@ -1,52 +1,70 @@
+ 
+Then Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the [`--use-free-memory-pct`](..//xtrabackup_bin/xbk_option_reference.md#use-free-memory-pct) option to run `--prepare`. The default value of the `--use-free-memory-pct` option is 0. This means that the option is disabled by default. If, for example, you set the option to 50, 50% of the free memory is used to `prepare` a backup. The maximum allowed value is 100.
 
-## Exapmple of `Smart memory estimation` usage
+Starting with Percona XtraBackup 8.0.32-26, you can enable or disable the memory estimation during the backup phase with the [`--estimate-memory`](..//xtrabackup_bin/xbk_option_reference.md#estimate-memory) option. The default value is `OFF`. Enable the memory estimation with  `--estimate-memory=ON`:
 
-Here is an example of how `Smart memory estimation` can improve the time spent on `prepare`.
+```{.bash data-prompt="$"}
+$ xtrabackup --backup --estimate-memory=ON --target-dir=/data/backups/
+```
 
-Let's say we run 3 backups, using sysbench with 16, 32, and 64 tables containing 1M rows each. The `--use-free-memory-pct` option is set to 50. This means that 50% of the free memory is used to `prepare` a backup. The backup is run on an ec2 c4.8xlarge instance (36 vCPU's / 60G memory / General Purpose SSD (gp2)). 
+In the `prepare` phase, enable the [`--use-free-memory-pct`](..//xtrabackup_bin/xbk_option_reference.md#use-free-memory-pct) option by specifying the percentage of free memory to be used to `prepare` a backup. The `--use-free-memory-pct` value must be bigger than 0.
 
-During each `--backup` the following sysbench is run:
+For example:
 
-  sysbench --db-driver=mysql --db-ps-mode=disable --mysql-user=sysbench --mysql-password=sysbench --table_size=1000000 --tables=${NUM_OF_TABLES} --threads=24 --time=0 --report-interval=1 /usr/share/sysbench/oltp_write_only.lua run
+```{.bash data-prompt="$"}
+$ xtrabackup --prepare --use-free-memory-pct=50 --target-dir=/data/backups/
+```
+
+## Example of Smart memory estimation usage
+
+Here is an example of how Smart memory estimation can improve the time spent on `prepare`.
+
+Let's say we run three backups using sysbench with 16, 32, and 64 tables containing 1M rows each. The `--use-free-memory-pct` option is set to 50. This means that 50% of the free memory is used to `prepare` a backup. The backup is run on an ec2 c4.8xlarge instance (36 vCPU's / 60G memory / General Purpose SSD (gp2)). 
+
+During each `--backup`, the following sysbench is run:
+
+```text
+sysbench --db-driver=mysql --db-ps-mode=disable --mysql-user=sysbench --mysql-password=sysbench --table_size=1000000 --tables=${NUM_OF_TABLES} --threads=24 --time=0 --report-interval=1 /usr/share/sysbench/oltp_write_only.lua run
+```
 
 The following table shows the backup details:
```

Comment:
(all measurements are in Gigabytes)
Remove the (GB) after each

Reply:
fixed